cards.py

```
@@ -1,95 +1,92 @@
+        symbol = CARD_RANKS[self.rank] + CARD_SUITS[self.suit]
+        return CARD_RANKS[self.rank] + CARD_SUITS[self.suit] + (" " if len(CARD_RANKS[self.rank]) == 1 else "")
+
+    def __repr__(self):
+        """
+        Returns a string representation
+        """
+        return CARD_RANKS[self.rank] + CARD_SUITS[self.suit]
+
+class Deck(list):
+    "A class representing a deck of cards"
+    def __init__(self, indeces=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)